okhttp-tests/src/test/java/okhttp3/CipherSuiteTest.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright (C) 2016 Google Inc.
```

Comment:
Google's preference would be an AUTHORS file (initially listing Square and Google) or alternatively to change the copyright on all files to "Copyright OkHttp Project Authors" since either of those would be easier to maintain and more indicative of others' contributions over time. Is that something you'd be comfortable with?